### PR TITLE
crossplane: fix the controller template for attribute-based APIs

### PR DESCRIPTION
**Pull request by muvaf**
Description of changes: Fixes the controller template for attribute-based APIs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: templates/crossplane/pkg/controller.go.tpl [hunk 1]
```diff
@@ -180,22 +180,30 @@ func newExternal(kube client.Client, client svcsdkapi.{{ .SDKAPIInterfaceTypeNam
 	e := &external{
 		kube:           kube,
 		client:         client,
-		{{- if or .CRD.Ops.ReadOne .CRD.Ops.GetAttributes .CRD.Ops.ReadMany }}
+		{{- if or .CRD.Ops.ReadOne }}
```
Comment (jaypipes) [inline templates/crossplane/pkg/controller.go.tpl:183]: I think this needs to be either this:

```
{{- if or .CRD.Ops.ReadOne .CRD.Ops.GetAttributes }}
```

or

```
{{- if .CRD.Ops.ReadOne }}
```

**Review by jaypipes — CHANGES_REQUESTED** [review]

File: templates/crossplane/pkg/controller.go.tpl (hunk 1 above)
Comment (muvaf) [reply templates/crossplane/pkg/controller.go.tpl]: Good catch, fixed! Thanks!

**Review by jaypipes — APPROVED** [review]
:+1:

**Pull request merged**